File: app/src/main/java/com/lumiyaviewer/lumiya/utils/IdentityMatrix.java
package com.lumiyaviewer.lumiya.utils;

import android.opengl.Matrix;

public class IdentityMatrix {
    private static final float[] matrix = new float[16];

    static {
        Matrix.setIdentityM(matrix, 0);
    }

    public static float[] getMatrix() {
        return matrix;
    }
}

File: app/src/main/java/com/lumiyaviewer/lumiya/utils/IdentityMatrix.kt
package com.lumiyaviewer.lumiya.utils

import android.opengl.Matrix

open class IdentityMatrix {
    companion object {
        private val matrix = FloatArray(16).also { Matrix.setIdentityM(it, 0) }
        @JvmStatic fun getMatrix(): FloatArray = matrix
    }
}
